backend/src/controllers/orders-controller.ts

```diff
@@ -0,0 +1,74 @@
+import { Request, Response } from 'express';
+import OrderItem, { IOrderItem } from '../models/order-item';
+import {validationResult} from "express-validator";
+import Order, {IOrder} from "../models/order";
+import { ObjectId } from 'mongodb';
+
+export const createOrder = async (req: Request, res: Response): Promise<void> => {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+        console.log("Validation errors", errors);
+        res.status(400).json({ errors: errors.array() });
+    }
+    try {
+        const { orderId, itemId, size, numOrdered } = req.body;
+        const orderItem: IOrderItem = new OrderItem({
+            orderId,
+            itemId,
+            size,
+            numOrdered,
```

Comment:
Use `Order` instead of `OrderItem`:

![image](https://github.com/miguelaenlle/merch-order-request-form/assets/66172204/e545d910-4846-4442-ad9c-f681808c2318)

Ensure fields line up too.